reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+
+See [the NATS website](https://nats.io/).
+
+<a name="platform"></a>
+### Platform
+
+An abstraction layer that is provided by Anbox to access the hardware resources of the host system from the Android system. Anbox Cloud supports three platforms: null (without rendering), webrtc (WebRTC) and swrast (software rendering).
+
+See https://discourse.ubuntu.com/t/anbox-platforms/18733.
+
+<a name="prometheus"></a>
+### Prometheus
+
+An open-source application used for event monitoring and alerting, which records real-time metrics about system events.
+
+See [the Prometheus website](https://prometheus.io/).
+
+<a name="raw-container"></a>
+### Raw container
+
+A container that is created when launching an image. It runs the full Android system, without any additional apps installed.
+
+See [Application containers vs. raw containers](https://discourse.ubuntu.com/t/about-containers/17763#application-vs-raw).
+
+<a name="regular-container"></a>
+### Regular container
+
+A container that is launched from either an application or an image. It exists until it is deleted.
+
+See https://discourse.ubuntu.com/t/about-containers/17763.
+
+<a name="scrcpy"></a>
+### Scrcpy
+
+An open-source screen mirroring application that allows displaying and controlling Android devices from a desktop computer.
+
+See [the scrcpy project on GitHub](https://github.com/Genymobile/scrcpy).
+
+<a name="session"></a>
+### Session
+
+The interaction between a streaming client and the application container during streaming. A session contains, among other information, user data and application information and provides an entry point for both the client and the container to start the signalling process.
+
+See https://discourse.ubuntu.com/t/about-application-streaming/17769.
+
+<a name="snap"></a>
+### Snap
+
+A software package for a desktop, cloud or IoT application that is easy to install, secure, cross‐platform and dependency‐free.
+
+See [the Snapcraft website](https://snapcraft.io/).
+
+<a name="swrast"></a>
+### Software Rasterization (swrast)
+
+An LLVMpipe-based software rendering platform that is useful for visual tests but does not provide audio input/output.
+
+See https://discourse.ubuntu.com/t/anbox-platforms/18733.
+
+<a name="stream-agent"></a>
+### Stream agent
+
+The software running on a server equipped with a GPU connected to Anbox Cloud, which serves as an entry point that the stream gateway can connect to.
+
+See https://discourse.ubuntu.com/t/about-application-streaming/17769.
+
+<a name="stream-gateway"></a>
+### Stream gateway
+
+The central component that connects clients with stream agents. Its role is to choose the best possible region depending on the user location and server capacities.
+
+See https://discourse.ubuntu.com/t/about-application-streaming/17769.
+
+<a name="streaming-stack"></a>
+### Streaming stack
+
+A collection of components designed to run containers on GPU-equipped machines and stream their visual output to clients via WebRTC.
```

Comment:
Not necessarily GPU equipped machines. It can run on machines without GPUs too using software rendering